sycl/include/CL/sycl/INTEL/esimd/detail/esimd_types.hpp

```diff
@@ -209,7 +209,7 @@ template <typename U> constexpr bool is_type() { return false; }
 template <typename U, typename T, typename... Ts> constexpr bool is_type() {
   using UU = typename detail::remove_const_t<U>;
   using TT = typename detail::remove_const_t<T>;
-  return std::is_same<UU, TT>::value || is_type<UU, Ts...>();
+  return sycl::detail::is_same_v<UU, TT> || is_type<UU, Ts...>();
```

Comment:
```suggestion
  return csd::is_same_v<UU, TT> || is_type<UU, Ts...>();
```

Reply:
Done, thanks for the suggestion. I've also applied the change to other `esimd/` files.